### PR TITLE
Add missing KernelGen labels to 4 operators in operators.yaml

### PR DESCRIPTION
**Pull request by Schopenhauer-loves-Hegel**
## Summary
- Add missing `KernelGen` label to 4 operators in `conf/operators.yaml` that were introduced via KernelGen PRs but had the label omitted:
  - `is_all_true` — 【KernelGen】Add _is_all_true operator (#1708)
  - `diff` — 【KernelGen】Add diff operator (#1730)
  - `margin_ranking_loss` — feat: add 12 experimental operators (Group 2) generated by KernelGen
  - `special_log_softmax` — [KernelGen][Nvidia] Add special_log_softmax operator with Triton kernel

## Methodology
Identified by cross-referencing the merge PR history of each operator's implementation file (`git log --diff-filter=A`) against the labels in `conf/operators.yaml`. Out of 175 operators checked, only these 4 had KernelGen-originated PRs but were missing the label.

## Test plan
- [ ] Verify `conf/operators.yaml` passes YAML validation
- [ ] No code changes — label-only metadata update

### CODE REVIEW

File: conf/operators.yaml [hunk 1]
```diff
@@ -7029,6 +7032,8 @@ ops:
       Applies the log of the softmax function along a given dimension.
     for:
       - special.log_softmax
+    labels:
+      - KernelGen
```
Comment (tengqm) [inline conf/operators.yaml:7038]: The metadata for this operator is incomplete, right?

Comment (lukatao) [reply conf/operators.yaml:7038]: yes, this operator was added by https://github.com/flagos-ai/FlagGems/pull/3803 from kernelgen. the main branch lacks the corresponding label

Comment (tengqm) [reply conf/operators.yaml:7038]: I mean, this operator doesn't have `kind` or `stages` information.

**Comment by lukatao** [comment]
Fixed in 85d0d8f. Added missing `kind: Math` and `stages: alpha '5.4'` fields, plus `aten` and `nn.functional` labels for consistency with other special.* operators.

**Review by tengqm — APPROVED** [review]
